nifi-assembly/pom.xml

```diff
@@ -470,6 +470,15 @@ language governing permissions and limitations under the License. -->
     </properties>
     <profiles>
         <profile>
+            <id>mapr</id>
```

Comment:
this profile is used under to customise `bootstrap.properties` to include `-Djava.security.auth.login.config=/opt/mapr/conf/mapr.login.conf`

AFAIK it is only required by MapR. To the best of my knowledge CDH and HDP do not require anything similar.

But you have a point, perhaps we could skip the profile could by simply referring people to use the property directly.

Having said that I think that telling people to use using `-Dnifi.hadoop.distro.params="-Djava.security.auth.login.config=/opt/mapr/conf/mapr.login.conf"` would look quite ugly and confusing when compared with `-Pmapr`